tasks/firewall.yml

```diff
@@ -0,0 +1,29 @@
+---
+- block:
+    - name: Add tcp ports to logging_firewall_ports
+      set_fact:
```

Comment:
Since you are using `set_fact`, you'll need to ensure that `logging_firewall_ports` is empty before this task, either by resetting it before this task, or resetting it after the task that calls the firewall role